template_feed/Microsoft.DotNet.Common.ProjectTemplates.10.0/content/MSTest-CSharp/.template.config/template.json

```diff
@@ -117,8 +117,8 @@
       "defaultValue": "VSTest",
       "choices": [
         {
-          "choice": "MSTest",
-          "description": "Use MSTest Runner (Microsoft.Testing.Platform)"
+          "choice": "Microsoft.Testing.Platform",
```

Comment:
I think there are bunch of `#if (TestRunner == "MSTest")` in template content that needs to be updated as well?